src/Php80/Php80.php

```diff
@@ -18,12 +18,35 @@
  */
 final class Php80
 {
-    public static function fdiv($dividend, $divisor)
+    public static function fdiv($dividend, $divisor): float
     {
         $dividend = self::floatArg($dividend, __FUNCTION__, 1);
         $divisor = self::floatArg($divisor, __FUNCTION__, 2);
 
-        return (float) @($dividend / $divisor);
+        return @($dividend / $divisor);
+    }
+
+    public static function get_debug_type($value): string
+    {
+        switch (true) {
+            case null === $value: return 'null';
+            case \is_bool($value): return 'bool';
+            case \is_string($value): return 'string';
+            case \is_array($value): return 'array';
+            case \is_int($value): return 'int';
+            case \is_float($value): return 'float';
+            case \is_object($value): break;
+            case $value instanceof \__PHP_Incomplete_Class: return '__PHP_Incomplete_Class';
+            default: return 'resource';
+        }
+
+        $class = \get_class($value);
+
+        if ("\0" !== ($class[15] ?? '')) {
```

Comment:
I guess that `15` magic number is for `\strlen('class@anonymous')`? Will it need to be changed after php/php-src#5153?